styles/normalize.css

```diff
@@ -1,10 +1,349 @@
-@CHARSET "UTF-8";
-h1,
-h2 {
-    background: url(../images/gradient.png) no-repeat;
-    background-size: 100% 100%;

```

Comment:
Your normalize.css file had the incorrect content.  It should have the content found  in https://necolas.github.io/normalize.css/